src/index.ts

```diff
@@ -721,44 +753,48 @@ export class SunsynkPowerFlowCard extends LitElement {
                                   class="${additional_aux_load === 2 ? '' : 'st12'}"/>
                             <text id="duration" x="132" y="352" class="${font !== true ? 'st14' : 'st4'} left-align"
```

Comment:
need to debug why this is not working lines 754-773

Reply:
OK the problem is 

let battery_energy = Number.isNaN(this.toNum(battenergy.state)) ? config.battery.energy : this.toNum(battenergy.state);

Reply:
Will quickly have a look, apologies, never saw it working on my end as I had a workaround for my lead acids, lithium will be here this week.